File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListView/PopulatingData/BindingDataSourcePage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.ListView.PopulatingData
{
    public partial class BindingDataSourcePage : UserControl
    {
        public BindingDataSourcePage()
        {
            InitializeComponent();
        }

        private void BindingDataSourcePage_Load(object sender, EventArgs e)
        {
            // Fill up adapter, that is DataSources for the ListView, with data of Database Customer table.
            this.customersTableAdapter.Fill(this.northwindDataSet.Customers);
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListView/PopulatingData/BindingDataSourcePage.vb
Namespace CompanionKit.Controls.ListView.PopulatingData

	Public Class BindingDataSourcePage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()



		End Sub

        Private Sub BindingDataSourcePage_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
            ' Fill up adapter, that is DataSources for the ListView, with data of Database Customer table.
            Me.CustomersTableAdapter.Fill(Me.NorthwindDataSet.Customers)
        End Sub
	End Class

End Namespace